ush/config.sh.RRFS_ens

```diff
@@ -48,6 +48,9 @@ MODEL="RRFSE"
 RUN="RRFS_ens"
 COMINgfs=""
 
+#EXTRN_MDL_SYSBASEDIR_LBCS="/mnt/lfs4/HFIP/gsihyb/Chunhua.Zhou/data/GEFS/public/pgrb2.merged"
+#EXTRN_MDL_SYSBASEDIR_ICS="/mnt/lfs4/HFIP/gsihyb/Chunhua.Zhou/data/HRRRE"
+
```

Comment:
It would be good to take these paths out before merging.

Reply:
Thank you for approving the PR and move things forward. I will remove these two lines.